src/Definition.php

```diff
@@ -89,4 +89,11 @@ class Definition
      * @var string
      */
     public $documentation;
+
+    /**
+     * Parameters array (for methods and functions), for use in textDocument/signatureHelp
+     *
+     * @var string[]
```

Comment:
You want me to build the objects here? Won't that use more memory than plain strings, which are converted to ParameterInformation when needed?